gems/i18n/CVE-2014-10077.yml

```diff
@@ -0,0 +1,18 @@
+---
+gem: i18n
+cve: CVE-2014-10077
+url: https://github.com/svenfuchs/i18n/pull/289
+title: i18n Gem for Ruby lib/i18n/core_ext/hash.rb Hash#slice() Function Hash Handling DoS
+date: 2014-09-27
+description: |
+  i18n Gem for Ruby contains a flaw in the Hash#slice() function in
+  lib/i18n/core_ext/hash.rb that is triggered when calling a hash when
+  :some_key is in keep_keys but not in the hash. This may allow an attacker
+  to cause the program to crash.
+
+patched_version:
```

Comment:
Should this not be `patched_versions` plural? I'm seeing this CVE breaking when I have `1.1.1` installed.

Reply:
... this is now fixed in https://github.com/rubysec/ruby-advisory-db/commit/25eb46646c70315ee35d580d8be0af2e8db283a6 - thanks! :D